pom.xml

```diff
@@ -21,7 +21,7 @@
     <project_name>fcrepo-transform</project_name>
     <project_organization>fcrepo4-exts</project_organization>
     <ldpath.version>3.2.1</ldpath.version>
-    <jersey.version>2.13</jersey.version>
+    <jersey.version>2.22.2</jersey.version>
```

Comment:
If the parent pom is `fcrepo` wouldn't this version come from there? I.e. wouldn't it be better to simply remove this line?